app/snippet.rb

```diff
@@ -0,0 +1,10 @@
+class Snippet
+  def self.generate(html)
+    html
+      .split('</h1>')[1..-1].join # make sure we skip anything before <h1>
+      .gsub(/(\<h.+\>)/, "") # remove headings
```

Comment:
The entire header. Sometimes pages begin with a header (https://docs.publishing.service.gov.uk/manual/deploying.html). I think it's clearer if we skip that. Also avoids a weird sentence with missing punctuation. What would you do for that?